### ISSUE TITLE
Module loading and start is broken

### ISSUE THREAD

**Issue by mohierf** [Alignak-monitoring/alignak#447]
The daemon modules loading and start is broken:
**Receiver example:**
```
[2016-10-12 06:22:39 CEST] INFO: [alignak.daemon] Loading modules...
[2016-10-12 06:22:39 CEST] INFO: [alignak.modulesmanager] Module alignak_module_nsca (nsca) imported.
[2016-10-12 06:22:39 CEST] ERROR: [alignak.modulesmanager] Module nsca properties: {'daemons': ['receiver'], 'phases': ['running'], 'type': 'nsca', 'external': True}
[2016-10-12 06:22:39 CEST] INFO: [alignak] Get a NSCA receiver module for plugin nsca
[2016-10-12 06:22:39 CEST] INFO: [alignak] [NSCA] configuration, allowed hosts : ''(15667), buffer length: 4096, payload length: -1, encryption: 0, max packet age: 60, check future packet: True, backlog: 10
[2016-10-12 06:22:39 CEST] ERROR: [alignak.modulesmanager] Module nsca intance: {'illegal_char': <_sre.SRE_Pattern object at 0x7f0b253704c8>, 'process': None, 'from_q': None, 'max_packet_age': 60, 'port': 15667, 'check_future_packet': True, 'phases': ['running', None], 'myconf': <module module=alignak_module_nsca alias=nsca />, 'buffer_length': 4096, 'loaded_into': 'unknown', 'output_decoding': 'UTF-8', 'props': {'daemons': ['receiver'], 'phases': ['running', None], 'type': 'nsca', 'external': True}, 'init_try': 0, 'rng': <random.Random object at 0x1895e20>, 'host': '', 'password': u'helloworld', 'properties': {'daemons': ['receiver'], 'phases': ['running', None], 'type': 'nsca', 'external': True}, 'is_external': True, 'name': u'nsca', 'to_q': None, 'payload_length': -1, 'modules': [], 'interrupted': False, 'encryption_method': 0, 'backlog': 10}
[2016-10-12 06:22:39 CEST] ERROR: [alignak.modulesmanager] Module nsca instance: <alignak_module_nsca.nsca.NSCA_receiver object at 0x7f0b24086210>
[2016-10-12 06:22:39 CEST] CRITICAL: [alignak.daemon] I got an unrecoverable error. I have to exit.
[2016-10-12 06:22:39 CEST] CRITICAL: [alignak.daemon] You can get help at https://github.com/Alignak-monitoring/alignak
[2016-10-12 06:22:39 CEST] CRITICAL: [alignak.daemon] If you think this is a bug, create a new ticket including details mentioned in the README
[2016-10-12 06:22:39 CEST] CRITICAL: [alignak.daemon] -----
[2016-10-12 06:22:39 CEST] CRITICAL: [alignak.daemon] Back trace of the error: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemons/receiverdaemon.py", line 407, in main
    self.setup_new_conf()
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemons/receiverdaemon.py", line 288, in setup_new_conf
    self.do_load_modules(self.modules)
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemon.py", line 382, in do_load_modules
    self.modules_manager.load_and_init(mod_confs)
  File "/usr/local/lib/python2.7/dist-packages/alignak/modulesmanager.py", line 100, in load_and_init
    self.get_instances()
  File "/usr/local/lib/python2.7/dist-packages/alignak/modulesmanager.py", line 244, in get_instances
    if not inst.external and not self.try_instance_init(inst):
AttributeError: 'NSCA_receiver' object has no attribute 'external'
```

**Broker example:**
```
[2016-10-12 06:26:08 CEST] ERROR: [alignak.modulesmanager] Module logs instance: <alignak_module_logs.logs.MonitoringLogsCollector object at 0x7f84fdcba7d0>
[2016-10-12 06:26:08 CEST] CRITICAL: [alignak.daemon] I got an unrecoverable error. I have to exit.
[2016-10-12 06:26:08 CEST] CRITICAL: [alignak.daemon] You can get help at https://github.com/Alignak-monitoring/alignak
[2016-10-12 06:26:08 CEST] CRITICAL: [alignak.daemon] If you think this is a bug, create a new ticket including details mentioned in the README
[2016-10-12 06:26:08 CEST] CRITICAL: [alignak.daemon] -----
[2016-10-12 06:26:08 CEST] CRITICAL: [alignak.daemon] Back trace of the error: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemons/brokerdaemon.py", line 886, in main
    self.setup_new_conf()
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemons/brokerdaemon.py", line 670, in setup_new_conf
    self.do_load_modules(self.modules)
  File "/usr/local/lib/python2.7/dist-packages/alignak/daemon.py", line 382, in do_load_modules
    self.modules_manager.load_and_init(mod_confs)
  File "/usr/local/lib/python2.7/dist-packages/alignak/modulesmanager.py", line 100, in load_and_init
    self.get_instances()
  File "/usr/local/lib/python2.7/dist-packages/alignak/modulesmanager.py", line 244, in get_instances
    if not inst.external and not self.try_instance_init(inst):
AttributeError: 'MonitoringLogsCollector' object has no attribute 'external'

```

---
**Note:** this exception is actually occuring in the Travis build of the clean-pr-logs branch .....

**Comment by mohierf** [comment]
The modules do not receive their own configuration parameters!

In my configuration, I defined an NSCA module for the receiver and a Logs module for the broker. The Logs module is receiving the parameters of the NSA module and the receiver do not have any module !

```
[2016-10-12 07:31:17 CEST] INFO: [alignak.module.logs] Get an instance of <module module=alignak_module_logs alias=logs />
[2016-10-12 07:31:17 CEST] INFO: [alignak.module.logs] Get an instance of alignak_module_logs: {'max_packet_age': u'', 'port': u'', 'check_future_packet': u'', 'module_alias': u'logs', 'use': [], 'uuid': u'ea9964eabee444ce8a05b802f5b3d7ae', 'log_dir': u'/usr/local/var/log/alignak', 'python_name': u'alignak_module_logs', 'username': u'', 'definition_order': 100, 'tags': set([]), 'host': u'', 'configuration_warnings': [u'Guessing the property log_dir type because it is not in Module object properties'], 'password': u'', 'properties': {'daemons': ['broker'], 'phases': ['running'], 'type': 'logs', 'external': True}, 'imported_from': u'/usr/local/etc/alignak/arbiter/modules/mod-logs.cfg:4', 'api_url': u'', 'configuration_errors': [], 'name': u'', 'payload_length': u'', 'register': True, 'modules': [], 'encryption_method': u''}
```


```
[2016-10-12 07:31:17 CEST] INFO: [alignak.daemons.receiverdaemon] We have our schedulers:
[2016-10-12 07:31:17 CEST] INFO: [alignak.daemons.receiverdaemon]  - scheduler-master 
[2016-10-12 07:31:17 CEST] INFO: [alignak.daemon] Loading modules...
[2016-10-12 07:31:17 CEST] INFO: [alignak.daemon] I do not have any module
```

**Comment by mohierf** [comment]
Perharps something related with #340

**Comment by mohierf** [comment]
Closed with #453
